src/EnergyPlus/DataHVACGlobals.hh

```diff
@@ -184,39 +178,21 @@ namespace HVAC {
         "InterlockedWithMechanicalCooling",
     };
 
-    //    enum class UnitarySysType
-    //    {
-    //        Invalid = -1,
-    //        Furnace_HeatOnly,
-    //        Furnace_HeatCool,
-    //        HeatOnly,
-    //        HeatCool,
-    //        HeatPump_AirToAir,
-    //        HeatPump_WaterToAir,
-    //        AnyCoilType,
-    //        Num
-    //    };
-    //
-    //    static constexpr std::array<std::string_view, (int)UnitarySysType::Num> unitarySysTypeNames = {
-    //        "AirLoopHVAC:Unitary:Furnace:HeatOnly",
-    //        "AirLoopHVAC:Unitary:Furnace:HeatCool",
-    //        "AirLoopHVAC:UnitaryHeatOnly",
-    //        "AirLoopHVAC:UnitaryHeatCool",
-    //        "AirLoopHVAC:UnitaryHeatPump:AirToAir",
-    //        "AirLoopHVAC:UnitaryHeatPump:WaterToAir",
-    //        "AirLoopHVAC:UnitarySystem"
-    //    };
-
-    // parameters describing unitary systems
-    int constexpr NumUnitarySystemTypes(7);
-    // Furnace/Unitary System Types
-    int constexpr Furnace_HeatOnly(1);
-    int constexpr Furnace_HeatCool(2);
-    int constexpr UnitarySys_HeatOnly(3);
-    int constexpr UnitarySys_HeatCool(4);
-    int constexpr UnitarySys_HeatPump_AirToAir(5);
-    int constexpr UnitarySys_HeatPump_WaterToAir(6);
-    int constexpr UnitarySys_AnyCoilType(7);
+    enum class UnitarySysType
```

Comment:
CustomCheck is not happy with this enumeration. I'm sorry CustomCheck, but this is just a bad enumeration. Deal.

Reply:
So it looks like it's actually just one entry causing the problem:

```
>>> import difflib
>>> difflib.get_close_matches('UnitarySysType', ['Furnace_HeatOnly', 'Furnace_HeatCool', 'Unitary_HeatOnly', 'Unitary_HeatCool', 'Unitary_HeatPump_AirToAir', 'Unitary_HeatPump_WaterToAir', 'Unitary_AnyCoilType'], cutoff=0.7)
['Unitary_AnyCoilType']
```

And it **barely** gets flagged.  If I set the cutoff to just 0.73 (0.03 higher), it won't get flagged:

```
>>> difflib.get_close_matches('UnitarySysType', ['Furnace_HeatOnly', 'Furnace_HeatCool', 'Unitary_HeatOnly', 'Unitary_HeatCool', 'Unitary_HeatPump_AirToAir', 'Unitary_HeatPump_WaterToAir', 'Unitary_AnyCoilType'], cutoff=0.73)
[]
```

Either way, I'll add it to the ignore list to get custom check passing.